cmd/gitannex/gitannex.md

```diff
@@ -0,0 +1,52 @@
+The gitannex command, aka "garrgoyle", enables git-annex to store and retrieve
+content from an rclone remote. It expects to be run by git-annex, not directly
+by users. It is an "external special remote program" as defined by git-annex's
+documentation.
+
+Installation on Linux
+---------------------
+
+1. Create a symlink and ensure it's on your PATH. For example:
+
+        ln -s "$(realpath rclone)" "$HOME/bin/git-annex-remote-rclone-goyle"
+
+2. Add a new external remote to your git-annex repo.
+
+   The new remote's type should be "rclone-goyle". If it is on your PATH,
```

Comment:
Could the type just be `rclone` rather than `rclone-goyle`? As someone who knows very little about the protocols involved, the word `goyle` means nothing to me - is it important?

Reply:
So if the name were just "rclone", then we would have a name collision with <https://github.com/git-annex-remote-rclone/git-annex-remote-rclone>. I'm working on changing the type to "rclone-builtin" right now, which seems reasonably self explanatory.

Back when I was thinking this would be a standalone command, rather than baked into rclone, I picked a globally-unique suffix that also made a semi-pun (git-annex-remote-rclone-goyle ==> "garrgoyle").